### ISSUE TITLE
German quotes are broken

### ISSUE THREAD

**Issue by marekdedic** [mvriel/TextToLatex#3]
In german (and a few other languages) the correct way to display (double) quotes is such that the first quotes are on the bottom like this : ,,text" (I am using commas here, but it shouldn't be commas...) . This utility breaks that by replacing `"text"` by ``` ``text''``` and so the quotes are always on the top.

Suggested solution: replace `"text"` by `\enquote{text}`. This solution needs the csquotes package, but quotes are then automatically rendered correctly based on the language settings in the document (babel or polyglossia).

**Comment by cebe** [comment]
Commas are not the correct quotes to use but `` "` `` and `` "' `` should be used instead. see https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Kodierung
These might still get broken by the implementation though, have not tested it.

**Comment by marekdedic** [comment]
No, no, I meant I am using commas here in Github issues because you can't do proper german quotes here :)
I know about the `` "` `` and `"'` option, but that would make it so there are german quotes for everyone. the `\enquote{}` command does english or german quotes based on your languge settings
